src/Inkluzitron/Modules/Vote/VotegagModule.cs

```diff
@@ -0,0 +1,210 @@
+        public async Task StartVoteAsync([Name("uživatel")] IUser user, [Name("minut")] int minutes = 30)
+        {
+            var minMinutes = VoteSettings.MuteMinutesMin;
+            var maxMinutes = VoteSettings.MuteMinutesMax;
+
+            if (minutes < minMinutes || minutes > maxMinutes)
+            {
+                await ReplyAsync($"Doba umlčení musí být v rozsahu {minMinutes}-{maxMinutes} minut.");
+                return;
+            }
+
+            if (Context.Guild.Id != BotSettings.HomeGuildId)
+            {
+                await ReplyAsync("Tento příkaz lze spustit jen na hlavním serveru.");
+                return;
+            }
+
+            if (user.IsBot)
+            {
+                await ReplyAsync("Nemůžeš umlčet bota.");
+                return;
+            }
+
+            var guildUser = Context.Guild.GetUser(user.Id);
+
+            if (guildUser == null)
+            {
+                await ReplyAsync($"{user.GetDisplayName()} není na tomto serveru.");
+                return;
+            }
+
+            var userDb = await UsersService.GetOrCreateUserDbEntityAsync(user);
+
+            var mutedGendered =
+                userDb.Gender == Gender.Male ? "umlčený" :
+                userDb.Gender == Gender.Female ? "umlčená" :
+                "umlčený/á";
+
+            var wasGendered =
+                userDb.Gender == Gender.Male ? "byl" :
+                userDb.Gender == Gender.Female ? "byla" :
+                "byl/a";
+
+            if (userDb.MutedUntil != null && userDb.MutedUntil > DateTime.Now)
+            {
+
+                await ReplyAsync($"{user.GetDisplayName()} již je {mutedGendered} do {userDb.MutedUntil?.ToString("HH:mm")}. Nelze tedy zahájit nové hlasování.");
+                return;
+            }
+
+            if(Cache.TryGetValue<DateTime>(CreateCacheKey(user.Id), out var voteEnd))
+            {
+                await ReplyAsync($"Již probíhá hlasování o umlčení {user.GetDisplayName()}. Do {voteEnd:HH:mm} tedy nelze zahájit nové hlasování.");
+                return;
+            }
+
+            var mentions = new AllowedMentions
+            {
+                MentionRepliedUser = false
+            };
+            mentions.UserIds.Add(guildUser.Id);
+
+            voteEnd = DateTime.Now.Add(VoteSettings.MuteVoteTimeout);
+            Cache.Set(CreateCacheKey(user.Id), voteEnd);
+
+            var minVotesCalculation = VoteSettings.MuteVoteMaxVotes -
+                Math.Pow((maxMinutes - minutes) / (double)(maxMinutes - minMinutes), 2)
+                * (VoteSettings.MuteVoteMaxVotes - VoteSettings.MuteVoteMinVotes);
+
+            var minVotes = (int)Math.Floor(minVotesCalculation);
+
+            var voteMessage = await ReplyAsync(GenerateVoteMessage(guildUser, minutes, minVotes, 0, voteEnd), allowedMentions:mentions);
+            await voteMessage.AddReactionAsync(VoteSettings.MuteReactionFor);
```

Comment:
existuje myslím AddReactionsAsync, kde jich jde nahrát víc jedním callem

Reply:
Ono je to doslova jedno. Protože jediný, co metoda `AddReactionsAsync` dělá je, že tam přidá cyklus.